Library/Homebrew/test/test_descriptions.rb

```diff
@@ -0,0 +1,35 @@
+require "testing_env"
```

Comment:
Please add `require "descriptions"` too so we can run e.g. `brew tests --only=descriptions`.